### PR TITLE
global: configurable aggregations

### PR DESCRIPTION
**Pull request by dinosk**
* ADDS loading of aggregation logic from entrypoints

Signed-off-by: Dinos Kousidis <konstantinos.kousidis@cern.ch>

### CODE REVIEW

File: invenio_stats/contrib/events.py [hunk 1]
```diff
@@ -23,12 +23,25 @@
...
-def register():
+def register_events():
     """Register sample events."""
     return [dict(event_type='file-download',
                  processor=EventsIndexer),
             dict(event_type='record-view',
                  processor=EventsIndexer)]
+
+
+def register_aggregations():
```
Comment (nharraud) [inline invenio_stats/contrib/events.py]: The file is name `events.py`. We should find a more appropriate name if we also put the aggregations there.

File: invenio_stats/contrib/events.py [hunk 2]
```diff
@@ -23,12 +23,25 @@
...
+    """Register sample aggregations."""
+    return [dict(event_type='file-download',
```
Comment (nharraud) [inline invenio_stats/contrib/events.py]: The name should not be the name of an event type. We could have multiple aggregations based on the same events.

File: invenio_stats/contrib/events.py (hunk 1 above)
Comment (dinosk) [inline invenio_stats/contrib/events.py]: Changed to registrations.py

File: invenio_stats/contrib/events.py (hunk 2 above)
Comment (dinosk) [inline invenio_stats/contrib/events.py]: Changed to aggregation_name

**Pull request merged**